src/base/extensions.ts

```diff
@@ -39,3 +43,28 @@ export interface IStartMenuSection {
 
   update?(): void;
 }
+
+
+export const EXTENSION_POINT_STARTMENU_DATASET = 'ordinoStartMenuDataset';
+
+export interface IStartMenuDatasetDesc extends IPluginDesc {
+  id: string;
+  name: string;
+  headerIcon: string;
+  viewId: string;
```

Comment:
```suggestion
  initialViewId: string;
```

I'm aware that we are using this `viewId` everywhere. However, I noticed that sometimes the variable name is a bit fuzzy. Hence, I suggest to rename it `initialViewId` to make clear that this is the first view of an analysis session. I'm open for other suggestions as well. Please remember, to rename it also in other places of the dataset cards and phovea.ts.